masterlist.yaml

```
@@ -18868,6 +18868,30 @@ plugins:
         util: '[SSEEdit v4.0.3](https://www.nexusmods.com/skyrimspecialedition/mods/164)'
         itm: 14
 
+  - name: 'Ryn''s Farms.esp'
+    url:
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/72305/'
+        name: 'Ryn''s Farms'
+    msg:
+      - <<: *patch3rdParty
+        subs:
+          - 'Skyrim Sewers'
+          - '[Patches for Ryn''s](https://www.nexusmods.com/skyrimspecialedition/mods/74233/)'
+        condition: 'active("Ryn''s Farms.esp") and not active("Ryn''s Farms - Skyrim Sewers.esp")'
```

The patch messages should only be displayed if the patched mod is actually active:

```suggestion
        condition: 'active("SkyrimSewers.esp") and not active("Ryn''s Farms - Skyrim Sewers.esp")'
```